src/components/controls.vue

```diff
@@ -154,9 +193,37 @@ export default {
     },
 
     onAddOverlay: function () {
-      alert('adding overlays in this demo is not implemented yet! :)')
+      if(this.addOverlyEnabled) {
+        this.dialog = true;
+      }
+      else {
+        alert('adding overlays in this demo is not implemented yet! :)')
+      }
+    },
+
+
+    visibilityIcon: function(val) {
+      return val ? 'mdi-eye' : 'mdi-eye-off';
     },
 
+    loadOverlay: function() {
```

Comment:
if `overlayUrl` does not contain a real url, or a url relative to the server running niivue (e.g. `mni152.nii.gz`) then it is still added to the layers list. It would be good to check if the user has input anything and if not, return without adding a layer. Currently even if `overlayURL` is empty, an empty layer drop-down row gets added to the layer list.

Maybe we can hold off on overlay changes though until @neurolabusc adds in the changes he mentioned recently. I suppose `niivue.js` should also have some logic to make sure an overlay can be added. For example, it wouldn't be useful to overlay one person's image onto another if the two were not already aligned somehow outside of niivue.